.gitignore

```diff
@@ -1,6 +1,6 @@
 .*
 !.gitignore
-gradle/
+!gradle/
```

Comment:
I previously thought we wouldn't want these in VCS, but just looked into it and found this:
>To make the Wrapper files available to other developers and execution environments you’ll need to check them into version control. 
https://docs.gradle.org/current/userguide/gradle_wrapper.html

So looks like this is the right approach.